static/app/components/profiling/FlamegraphWarnings.tsx

```diff
@@ -59,6 +59,6 @@ const Overlay = styled('div')`
   display: grid;
   grid: auto/50%;
   place-content: center;
-  z-index: ${p => p.theme.zIndex.modal};
+  z-index: 1;
```

Comment:
What happens if you just remove the z-index here? If we need to explicitly make it 1, let's use the constant here.

```suggestion
  z-index: ${p => p.theme.zIndex.initial};
```

Reply:
if you don't use a z-index then the message saying there is no samples or duration is 0 + Export Raw Profile button will be sent to the back and unclickable.

i've updated my code to use the theme value